src/strings.cpp

```diff
@@ -1618,6 +1618,12 @@ static char *FormatString(char *buff, const char *str_arg, StringParameters *arg
 				break;
 			}
 
+			case SCC_COLOUR: { // {COLOUR}
+				StringControlCode scc = (StringControlCode)(SCC_BLUE + args->GetInt32(SCC_COLOUR));
+				if (IsInsideMM(scc, SCC_BLUE, SCC_COLOUR) && (buff + Utf8CharLen(scc) < last)) buff += Utf8Encode(buff, scc);
```

Comment:
This relies on SCC_COLOUR being placed after SCC_BLACK. May be better as `SCC_BLACK + 1`? Though this does all rely on the colours being in specific order in the first place, so may not matter.